src/server/db/user.ts

```diff
@@ -3,6 +3,7 @@ import { Doc } from 'src/types/common'
 
 export interface User extends Doc {
   name: string
+  studentID?: number
```

Comment:
can you change the type to string? Although student id in our school is 8 digit number, it might vary if we run study outside or something.